charts/datadog/values.yaml

```diff
@@ -2911,3 +2912,39 @@ otelAgentGateway:
     # If true, checks OTel Collector config for filelog receiver and mounts additional volumes to collect containers
     # and pods logs.
     enabled: false
+
+  autoscaling:
```

Comment:
Can you add descriptions for all the configurable fields here? Sorry as I know there are a lot of fields, but we use this info to populate the description of configurable fields in the readme